src/constants/fonts/primary.ts

```diff
@@ -2,7 +2,7 @@ const PRIMARY_FONTS = {
   headerFont:
     "'nocturno', Georgia, TimesNewRoman, 'Times New Roman', Times, Baskerville, serif",
   baseFont:
-    "'larssiet', 'Avant Garde', Avantgarde, 'Century Gothic', CenturyGothic, AppleGothic, Verdana, sans-serif",
+    "'skintype-text', 'Avant Garde', Avantgarde, 'Century Gothic', CenturyGothic, AppleGothic, Verdana, sans-serif",
```

Comment:
Unrelated to this PR, Avant Garde/avantgarde font doesn't seem to be loading. If I inspect source on the live site right now and remove larssiet from the stack, I get Century Gothic. This could be an environment-specific issue, but there's also no other reference to Avant Garde here, including any import. Should we do something about that?